codebase/codebase_test.go

```diff
@@ -180,7 +184,7 @@ func (test *TestCodebaseRepository) TestListCodebases() {
 	// then
 	require.NoError(test.T(), err)
 	require.Len(test.T(), codebases, 1)
-	require.Equal(test.T(), fxt.Codebases[0].URL, codebases[0].URL)
+	require.Contains(test.T(), repoUrls, codebases[0].URL)
```

Comment:
shouldn't this be `Equal`?
require.Equal(test.T(), repoUrls[0], codebases[0].URL)

Reply:
yeah, that's how it was earlier, but something is causing the order of results to change and not be the same, that's why changed code from `Equal` to `Contains`